src/test/java/com/jalasoft/pivotal/steps/deleteAccountSteps.java

```diff
@@ -0,0 +1,107 @@
+package com.jalasoft.pivotal.steps;
+
+import com.jalasoft.pivotal.pages.Account.Account;
+import com.jalasoft.pivotal.pages.Account.AccountSettings;
+import com.jalasoft.pivotal.pages.Account.CreateNewAccountForm;
+import com.jalasoft.pivotal.pages.Account.UserAccounts;
+import com.jalasoft.pivotal.pages.Dashboard;
+import com.jalasoft.pivotal.pages.Header;
+import com.jalasoft.pivotal.pages.ProfileDropdown;
+import com.jalasoft.pivotal.pages.Signin;
+import com.jalasoft.pivotal.pages.project.MorePage;
+import com.jalasoft.pivotal.pages.project.ProjectDetails;
+import com.jalasoft.pivotal.pages.project.ProjectForm;
+import cucumber.api.java.en.*;
+import org.junit.Assert;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.lang.Thread.sleep;
+
+public class deleteAccountSteps {
+    private String projectName = "My Project Eval";
+    private String projectAccount = "account1";
+    private String projectPrivacy = "public";
+    private ProfileDropdown profileDropdown;
+    private UserAccounts accounts;
+    private Header header;
+    private Dashboard dashboard;
+    private ProjectForm projectForm;
+    private Account account;
+    private ProjectDetails projectDetails;
+    private CreateNewAccountForm accountForm;
+    private Map<String,String> projectData;
+    private  MorePage more;
+    private AccountSettings settings;
+
+    public deleteAccountSteps() {
+        init();
+    }
+
+    private void init(){
+        dashboard = new Dashboard();
+        projectForm = new ProjectForm();
+        projectData = new HashMap<>();
+        projectData.put("name", projectName);
+        projectData.put("account", projectAccount);
+        projectData.put("privacy", projectPrivacy);
+        projectDetails = new ProjectDetails();
+        header = new Header();
+        settings = new AccountSettings();
```

Comment:
Could you use dependency injection here to instantiate the classes ?

Reply:
done